hashicorp/boundary-ui
# Icu 11936 support node 20 stop support node 16

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-11936)

## Description

support node 20 stop support node 16



## How to Test

Update local node to version 20.*. You should be able to run test, lint, format, build, and start the app locally with no issues.
node v20.* will show the below error message due to our ember version. So far there have been no issues and the Github runners will continue to use node v18 for now.
`WARNING: Ember CLI v4.12.2 is not tested against Node v20.12.2. See "https://github.com/ember-cli/ember-cli/blob/master/docs/node-support.md" to find out which version of Node is best to use.`

## Checklist:
